Changing device orientation during splash causes exception

Changing device orientation during splash causes exception...

```
E/AndroidRuntime(16937): FATAL EXCEPTION: main                                                                                                        [2123/9394]
E/AndroidRuntime(16937): java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
E/AndroidRuntime(16937):         at org.ruboto.JRubyAdapter.runRubyMethod(JRubyAdapter.java:58)
...
E/AndroidRuntime(16937): Caused by: java.lang.reflect.InvocationTargetException
E/AndroidRuntime(16937):         at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime(16937):         at java.lang.reflect.Method.invoke(Method.java:511)
...
E/AndroidRuntime(16937): Caused by: org.jruby.embed.InvokeFailedException: java.lang.IllegalStateException: No activity
E/AndroidRuntime(16937):         at org.jruby.embed.internal.EmbedRubyObjectAdapterImpl.call(EmbedRubyObjectAdapterImpl.java:319)
...
E/AndroidRuntime(16937): Caused by: java.lang.IllegalStateException: No activity
E/AndroidRuntime(16937):         at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1044)
...
```

My workaround is to lock initial activity to portrait and block orientaton change events from calling `onCreate()`.

Change `<activity>` in `AndroidManifest.xml` :
_(Remove '!screenSize' for API < 13 ?)_

```
<activity
  android:label='MyActivity'
  android:name='MyActivity'
  android:screenOrientation='portrait'
  android:configChanges='orientation|screenSize' >
...
</activity>
```

This workaround works for me and does not affect my main activity's handling of orientation changes because my main activity is started later as in the workaround for #443.

For normal Ruboto activity start, this workaround would require dealing with orientation changes in `onConfigChanged()` instead of it restarting the activity.

However, this approach to handling configuration change is discouraged:
http://developer.android.com/guide/topics/resources/runtime-changes.html#HandlingTheChange

--- a/lib/ruboto/util/update.rb
+++ b/lib/ruboto/util/update.rb
@@ -302,7 +302,7 @@ module Ruboto
           end
 
           unless app_element.elements["activity[@android:name='org.ruboto.SplashActivity']"]
-            app_element.add_element 'activity', {'android:name' => 'org.ruboto.SplashActivity', 'android:exported' => 'false'}
+            app_element.add_element 'activity', {'android:name' => 'org.ruboto.SplashActivity', 'android:exported' => 'false', 'android:screenOrientation' => 'portrait'}
           end
 
           unless app_element.elements["activity[@android:name='org.ruboto.RubotoDialog']"]

